### ISSUE TITLE
Resize images

### ISSUE THREAD

**Issue by anonieee** [karatsuba/workflowy-link-preview#25]
Any plans to add a handle to make the images resizable?

**Comment by karatsuba** [comment]
hey @anonieee! thanks for reaching me out :)
Amazing idea! Such a resizable feature could be really useful!
\+ what do you think about image resizing in case of URL preview with title and description?

**Comment by anonieee** [comment]
Thanks for the reply
> what do you think about image resizing in case of URL preview with title and description
I'm not really sure about this one. Maybe just have resizing apply to only images and not URL previews.
Also what do you think of a global resizing slider vs. individual ones? Maybe a global one is easier to implement.

**Comment by karatsuba** [comment]
Yep, gonna add resizing to only images 👍 

What do you mean by **_global resizing slider_** ? Because my initial intention is to add resizing to all embedded images + preserve their modified size after.

**Comment by anonieee** [comment]
Yeah I was thinking of individual modifiers, but now that I think about it a global slider is way more convenient anyways
